usr.sbin/inetd/inetd.h

```diff
@@ -0,0 +1,136 @@
+#ifndef _INETD_H
```

Comment:
Copy the copyright from inetd.c to inetd.h since the code was lifted from there.